docs/deep-dives/desktop/desktop-pref-experiments.md

```diff
@@ -8,10 +8,7 @@ As of Firefox 107, Nimbus supports experiments that set preferences on Desktop.
 Unlike Normandy, Nimbus cannot set arbitrary preferences; instead, the
 preferences that may be set are determined by the feature manifest.
 
-Each variable in a Nimbus feature can set a single pref. Integer, string, and
-boolean typed variables are supported but JSON variables are not supported. If
-you want to set a JSON value to a pref, a string variable should be used and
-experiments should set the value to a JSON string.
+Each variable in a Nimbus feature can set a single pref.
```

Comment:
Since it's a new capability, maybe worth calling out that all types are supported (something like below, or include the list)?
```suggestion
Each variable in a Nimbus feature can set a single pref (of any type -- JSON variables support added 2024-04-04).
```